vbfml/scripts/err_analysis

```diff
@@ -72,13 +75,35 @@ def cli():
     required=True,
     help="Tag to identify the process.",
 )
+@click.option(
+    "-r1",
+    "--range-seq1",
+    required=False,
+    type=float,
+    default=0,
+    help="read_range from ... (adjust read_range in the sequence to allow to read from any event)",
+)
+@click.option(
+    "-r2",
+    "--range-seq2",
+    required=False,
+    type=float,
+    default=1.0,
+    help="read_range to ... (adjust read_range in the sequence to allow to read to any event)",
+)
```

Comment:
Looking at the default values here, is the most default thing we want to do is to predict on the whole dataset? I remember there was a comment on one of the group meetings that it would be nice to show predictions for the events which the CNN hasn't trained on. That's why initially I hard-coded a sequence range of (0.8,1.0).

That being said, I think this is a nice functionality and we can keep it. Just wanted to quickly ask you if you think default values are optimal. And could you remove the comment above the `read_range` definition as it is no longer relevant? You can replace it with something like:

```
# Get the read range from the command line, default is (X,Y)
read_range = (range_seq1, range_seq2) 
```

Reply:
Corrected.

My bad, I was doing some test on full data and changed it for simplicity, but forgot to change it back. thanks for that